server/update_branch.go

```diff
@@ -0,0 +1,45 @@
+package server
+
+import (
+	"context"
+	"strings"
+
+	"github.com/mattermost/mattermost-server/mlog"
+
+	"github.com/google/go-github/v28/github"
+)
+
+func (s *Server) handleUpdateBranch(eventIssueComment IssueComment) {
+	client := NewGithubClient(s.Config.GithubAccessToken)
+	prGitHub, _, err := client.PullRequests.Get(context.Background(), *eventIssueComment.Repository.Owner.Login, *eventIssueComment.Repository.Name, *eventIssueComment.Issue.Number)
+	if err != nil {
+		mlog.Error("Error getting the latest PR information from github", mlog.Err(err))
+		return
+	}
+	pr, err := s.GetPullRequestFromGithub(prGitHub)
+	if err != nil {
+		mlog.Error("Error Updating the PR in the DB", mlog.Err(err))
+		return
+	}
+
+	userComment := *eventIssueComment.Comment.User
+	if !s.checkUserPermission(userComment.GetLogin(), pr.RepoOwner) {
```

Comment:
If the PR is made from a fork this will fail, even though `userComment` might have access. Can we check `pr.MaintainerCanModify` (https://godoc.org/github.com/google/go-github/github#PullRequest) if the check fails?

Reply:
this is the same for other command and i've test that with a user that is not part of the org

Reply:
Sorry, I was not clear on this:
If the PR is created from a fork, we should check `pr.MaintainerCanModify` and post a message that the PR creator should allow this.